src/ui/sass/modules/_Results.scss

```diff
@@ -163,8 +163,8 @@
     border: $border-legacy;
 
     .yxt-Results-filters {
-      border-left: none;
-      border-right: none;
+      border-left: $border-legacy;
```

Comment:
I know this is being super nitty but didn't we explicitly set these to border-left/right: none? Did something change?

Reply:
like a few weeks ago

Reply:
Not sure, but tom approved so it seems not